src/strategies/total-axion-shares/index.ts

```diff
@@ -4,7 +4,7 @@ import { BigNumber } from '@ethersproject/bignumber';
 export const author = 'Axion Foundation';
 export const version = '0.2.0';
 
-const data_reader_address = '0x421456eFcEBf814975c8739CD613e5e7a954C474';
+const data_reader_address = '0x8458fe26CbF3B8295755654C02ABaDFB1d3CBCe6';
```

Comment:
https://etherscan.io/address/0x8458fe26CbF3B8295755654C02ABaDFB1d3CBCe6

Looks like the address is not a contract, maybe keep the older address and just make options.dataReader optional

Reply:
Axion has migrated to Polygon, so we would switch the space to Polygon as well. This data reader exists on Polygon.